make/config.mk

```diff
@@ -132,10 +132,19 @@ endif
 ARCH := $(shell uname -a)
 ifneq (,$(filter $(ARCH), armv6l armv7l powerpc64le ppc64le aarch64))
 	USE_SSE=0
+	USE_F16C=0
```

Comment:
Please print a message if this is getting disabled

Reply:
That's not the behavior of any make flag though. Cmake does do that. Also for those devices there, F16C instruction set is absent. So it's not really a choice

Reply:
Well this is basically the first case, right?